dataflow/src/org/checkerframework/dataflow/cfg/node/LambdaResultExpressionNode.java

```diff
@@ -0,0 +1,81 @@
+package org.checkerframework.dataflow.cfg.node;
+
+import com.sun.source.tree.ExpressionTree;
+import java.util.Collection;
+import java.util.Collections;
+import javax.lang.model.util.Types;
+import org.checkerframework.dataflow.util.HashCodeUtils;
+import org.checkerframework.javacutil.InternalUtils;
+
+/*>>>
+import org.checkerframework.checker.nullness.qual.Nullable;
+*/
+
+/**
+ * A node for the single expression body of a single expression lambda
+ *
+ * @author Lazaro Clapp
+ */
+public class LambdaResultExpressionNode extends Node {
+
+    protected ExpressionTree tree;
```

Comment:
Can you add some documentation what the tree and result represent, either here or, maybe better, with the public getter methods?

Reply:
Adding it to the methods, let me know if its clear enough (working on the code as I reply to the comments, will update the PR shortly).